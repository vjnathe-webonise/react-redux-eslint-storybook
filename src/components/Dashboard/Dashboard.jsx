import { Link } from 'react-router-dom';

export default function Dashboard() {
  return (
    <div>
      <h1>Home</h1>
      <nav>
        <Link to="/">Home</Link> | <Link to="user">Users</Link> | <Link to="login"> Login</Link>
      </nav>
    </div>
  );
}
