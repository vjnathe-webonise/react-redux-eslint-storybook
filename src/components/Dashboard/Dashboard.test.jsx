import renderer from 'react-test-renderer';
import Dashboard from './Dashboard';
import { BrowserRouter as Router } from 'react-router-dom';

it('Dashboard componenet renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Dashboard />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
