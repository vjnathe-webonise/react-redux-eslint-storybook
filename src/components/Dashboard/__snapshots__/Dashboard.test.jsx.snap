// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard componenet renders correctly 1`] = `
<div>
  <h1>
    Home
  </h1>
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
     | 
    <a
      href="/user"
      onClick={[Function]}
    >
      Users
    </a>
     | 
    <a
      href="/login"
      onClick={[Function]}
    >
       Login
    </a>
  </nav>
</div>
`;
